.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Not valid json

Reply:
This is what vscode generated automatically :thinking: 
I'll just remove it and also add an empty new line at the end

Reply:
I know, I think it's bad that they do it